src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -31,12 +31,12 @@ internal class FunctionExecutor : IFunctionExecutor
         private readonly IAsyncCollector<FunctionInstanceLogEntry> _functionEventCollector;
         private readonly ILogger _logger;
         private readonly ILogger _resultsLogger;
-        private readonly IJobInvoker _jobInvoker;
+        private readonly IExtensionRegistry _extensionRegistry;
```

Comment:
>IExtensionRegistry [](start = 25, length = 18)

We shouldn't be passing this object around - too high level (same objection I had to passing JobHostConfig or even JobHost).